### DISCUSSION TITLE
Can't going to main page in safe browser

### DISCUSSION THREAD

**Discussion by nadilperera3** [SafeExamBrowser/seb-mac#392] [General]
I did an exam a few weeks ago with Seb, and now I have tried to open back seb but its redirecting to that Exam site after exit by using password

**Comment by danschlet** [comment]
You mean the iOS version?

That exam set the (persistent/permanent) client settings of the SEB app installed on your device. That's not best practice, they should use exam settings (which are only active temporarily during an exam). But maybe they expect that you use SEB only with their exam site.

In the iOS version of SEB you can reset client settings in the iOS Settings app, scroll down to SEB, enable "Initiate Settings Reset" and return to/start the SEB app.

If SEB asks for the admin password, you can delete the SEB app and install it again.

In case you're using the macOS app, use `defaults delete org.safeexambrowser.SafeExamBrowser` in Terminal to reset its settings.

**Reply by nadilperera3** [reply]
Uploading 20240402_233259_1.mp4 …

Yeah, I'm using Mac. I can't access settings  because it's not opening the home page

**Comment by danschlet** [comment]
If you can't get out of SEB anymore, then [see our FAQ](https://safeexambrowser.org/faq/faq_en.html#MacOSX).

**Reply by nadilperera3** [reply]
Thanks problem solved
